openstack/compute/v2/flavors/results.go

```diff
@@ -46,9 +46,11 @@ type Flavor struct {
 	Swap int `json:"swap"`
 	// VCPUs indicates how many (virtual) CPUs are available for this flavor.
 	VCPUs int `json:"vcpus"`
+	// IsPublic indicates whether the flavor is public.
+	IsPublic bool `json:"is_public"`
 }
 
-func (r *Flavor) UnmarshalJSON(b []byte) error {
+func (f *Flavor) UnmarshalJSON(b []byte) error {
```

Comment:
@mfranc213 I'm so sorry. I don't know how I missed this before. `r` is actually correct here. See the Naming section in the [style guide](https://github.com/gophercloud/gophercloud/blob/master/STYLEGUIDE.md) for more info.